libs/galley-types/src/Galley/Types.hs

```diff
@@ -286,10 +288,16 @@ newtype Accept = Accept
 
 -- Members ------------------------------------------------------------------
 
+-- The semantics of the possible different values is entirely up to clients,
+-- the server will not interpret this value in any way.
+newtype MutedStatus = MutedStatus { fromMutedStatus :: Int32 }
+    deriving (Eq, Num, Ord, Show, FromJSON, ToJSON)
+
 data Member = Member
     { memId             :: !UserId
     , memService        :: !(Maybe ServiceRef)
-    , memOtrMuted       :: !Bool
+    , memOtrMuted       :: !Bool -- ^ DEPRECATED, remove it once enough clients use `memOtrMutedStatus`
+    , memOtrMutedStatus :: !(Maybe MutedStatus)
```

Comment:
Drop the `Maybe`?  If not, please explain difference between `Nothing` and `Just 0` in comment.

Reply:
The `Maybe` is here by design; if it's a nothing, then clients have not yet set this value which is the reason why we don't want to use a default; `Just 0` could mean `all message types muted` or it could also mean `no message types muted`.

Reply:
Ok, I believe you, just trying to understand: `int` in cassandra can be `NULL`, which means that all bits have their default, which we don't want to know.  Yes, that makes sense, thanks!